nav2_msgs/msg/CircleVO.msg

```diff
@@ -0,0 +1,6 @@
+std_msgs/Header header
```

Comment:
The naming here I think can be more clear potentionally for the polygon/circle.

`CircleObject` `PolygonObject` `CircleObstacle` `PolygonObstacle` `GenericObstacle`, any thoughts? `VO` I don't think is necessarily all that clear